Doras/src/app/roles/roles.component.html

```diff
@@ -6,54 +6,64 @@
           <table class="tableBand">
             <tr>
               <th class="th1" scope="col">
-              <h2 class="jobName">{{jobBandArray[i][0].bandName}}</h2>
-                <a *ngFor="let job1 of firstJob">
-                <h1 *ngIf="job1; else notShowBlock">{{job1.title}}</h1>
-                <div *ngIf="job1;">
-                <p></p>
-                <img title="Responsibilities" src='assets/responsibilities.png' onmouseover="this.src='assets/responsibilities_hover.png';" onmouseout="this.src='assets/responsibilities.png';"class="topLeftIcon" (click)="openDialog('responsibility', 'job1')"/>
-                <img title="Training" src='assets/training.png' onmouseover="this.src='assets/training_hover.png';" onmouseout="this.src='assets/training.png';" class="topRightIcon" (click)="openDialog('training', 'training1')"/>
-                <p></p>
-                <img title="Job Specification" src='assets/jobSpec.png' onmouseover="this.src='assets/jobSpec_hover.png';" onmouseout="this.src='assets/jobSpec.png';" class="topLeftIcon" (click)="openDialog('spec', 'spec1')"/>
-                <img title="Other Jobs in the Same Band" src='assets/sjsb.png' onmouseover="this.src='assets/sjsb_hover.png';" onmouseout="this.src='assets/sjsb.png';" class="topRightIcon" />
-                </div>
+              <mat-card>
```

Comment:
this whole page is heavily hardcoded! each mat-card should be dynamically generated